CGWork/iritSkel.cpp

```diff
@@ -235,6 +235,9 @@ bool CGSkelStoreData(IPObjectStruct *PObj)
 	childObject.CalcBoundingBox();
 	parentObject.ReCalcBoundingBox(childObject);
 
+	// set intuitive sensitivity
+	parentObject.setDefaultSensitivity();
```

Comment:
this gets called once for every child object in the irit file, should move this call to either the initialized check in OnDraw  or to CGSkelProcessIritDataFiles function which only gets called once for every irit file